### ISSUE TITLE
(SH & BT) shaking machine gun

### ISSUE THREAD

**Issue by bulldozerecske** [openmoh/openmohaa#166]
MG is shaking on the tripod.
![image](https://github.com/openmoh/openmohaa/assets/8028500/fb4b3341-77df-44d5-a5bb-c3754c2c1269)

and some errors related to the model:

```
Skeletor CacheAnimSkel: Could not open binary file models/weapons/mg42portable_placed_setup_viewmodel.skc
TIKI_InitTiki: Failed to load animation 'models/weapons/mg42portable_placed_setup_viewmodel.skc' at
TIKI_InitTiki: no 'idle' animation found, model bounds not set for models/weapons/mg42portable_placed_setup_viewmodel.tik
```

```
Tiki:LoadAnim Couldn't load models/weapons/mg42portable_placed_setup_viewmodel.skd
^~^~^ Model 'models/weapons/mg42portable_placed_setup_viewmodel.tik' has no skelmodel
^~^~^ Bad model name 'models/weapons/mg42portable_placed_setup_viewmodel.tik'
PortableTurret::CreateViewModel : Couldn't find turret view model tiki models/weapons/mg42portable_placed_setup_viewmodel.tik
CarryableTurret::Weapon::GiveStartingAmmoToOwner : Owner not found
```

**Comment by bulldozerecske** [comment]
Same issue on Stalingrad...
![image](https://github.com/openmoh/openmohaa/assets/8028500/5cc87ffc-a88a-4f2f-9ea8-4c1040a0a9bf)

**Comment by smallmodel** [comment]
Fixed in commit 74e8a38c00d5a7974a04131207826fc0508c528b
